.github/workflows/secrets-nats-kv.yml

```
@@ -218,15 +218,38 @@ jobs:
           generate_release_notes: true
           files: |
             ./artifacts/*
+  
+  # This step runs so we can have the artifacts available in the right place for the OCI step. We
+  # might just want to build the rest of the binaries in nix so we aren't building twice, but for now
+  # this will work
+  build-nix:
+    if: ${{ startsWith(github.ref, 'refs/tags/secrets-nats-kv-v') }}
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```

The `# vX.X.X` won't get added automatically with updates if it's not included, it would be nice to have for easier comparisons:
```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```